FrameworkSystem/Client/WebAppCompiler.py

```diff
@@ -235,7 +247,7 @@ def run(self):
     gLogger.verbose(" IN file written to %s" % inFile)
 
     cmd = ['sencha', '-sdk', self.__sdkPath, 'compile', '-classpath=%s' % ",".join(self.__classPaths),
-           '-debug=%s' % self.__debugFlag, 'page', '-yui', '-input-file', inFile, '-out', outFile]
```

Comment:
you don't want debug?

Reply:
It is supposed to be used for printing the debug messages, which does not help in production...